src/pcapi/domain/admin_emails.py

```diff
@@ -48,3 +50,18 @@ def send_payments_report_emails(
 def send_offer_creation_notification_to_administration(offer: Offer, author: User) -> bool:
     email = make_offer_creation_notification_email(offer, author)
     return mails.send(recipients=[settings.ADMINISTRATION_EMAIL_ADDRESS], data=email)
+
+
+def send_offer_creation_refusal_notification_to_administration(offer: Offer, author: User) -> bool:
```

Comment:
```suggestion
def send_offer_rejection_notification_to_administration(offer: Offer) -> bool:
```
On pourrait préciser ainsi ce nom de fonction.

J'aurais tendance à ne pas passer l'`author` à la fonction et déléguer à la fonction `make...()` la récupération de l'`author`

Reply:
:+1: